### PR TITLE
Hotfix email

### PR DESCRIPTION
**Pull request by wnowicki**
@EvanBarbour3 I think it will be good to have this in `master` don't you ?

### CODE REVIEW

**Comment by EvanBarbour3** [comment]
@sean

File: scripts/post-install.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+chmod -R 777 storage
```
Comment (srleach) [inline scripts/post-install.sh]: Unnecessary

Comment (srleach) [inline scripts/post-install.sh]: https://www.google.co.uk/url?sa=t&rct=j&q=&esrc=s&source=web&cd=2&cad=rja&uact=8&ved=0ahUKEwithsKy3YTOAhXHCsAKHXbrCEAQFggjMAE&url=https%3A%2F%2Fblackchickenhost.com%2Fwhmcs%2Fknowledgebase%2F71%2FWhy-You-Should-Never-Use-777-Permissions-Anywhere-Ever-Full-Stop-End-of-Story.html&usg=AFQjCNEX__VGVmgrjtJrnqzlVyna_ubI8w&bvm=bv.127521224,d.ZGg

Comment (srleach) [inline scripts/post-install.sh]: The problem was caused by the vendor:publish not having been run

**Pull request merged**